From: "Liu-Chan, Ming" <ming.liu-chan@digi.com>
Date: Mon, 18 May 2020 12:46:57 -0400
Subject: at: increased AT port probe attempts

ModemManager sometimes didn't receive probe response when
switching from ATT to Verizon FW on LM960. Then, the modem
became inaccessible. ModemManager was probing the AT port
up to 3 times and then marked the AT port unavailable when
no response received.
We saw ModemManger received probe reply at 4th probe on LM960.
---
 src/mm-port-probe.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/mm-port-probe.c b/src/mm-port-probe.c
index 0c1edef1..c69f0ef4 100644
--- a/src/mm-port-probe.c
+++ b/src/mm-port-probe.c
@@ -1053,6 +1053,9 @@ serial_probe_at (MMPortProbe *self)
 }
 
 static const MMPortProbeAtCommand at_probing[] = {
+    { "AT",  3, mm_port_probe_response_processor_is_at },
+    { "AT",  3, mm_port_probe_response_processor_is_at },
+    { "AT",  3, mm_port_probe_response_processor_is_at },
     { "AT",  3, mm_port_probe_response_processor_is_at },
     { "AT",  3, mm_port_probe_response_processor_is_at },
     { "AT",  3, mm_port_probe_response_processor_is_at },
